moped-editor/src/views/projects/projectView/ProjectComponents/TheMap.js

```diff
@@ -1,21 +1,25 @@
 import { useState, useMemo } from "react";
 import MapGL from "react-map-gl";
 import { cloneDeep } from "lodash";
-import FeaturePopup from "./FeaturePopup";
```

Comment:
Commenting out the feature popup for now since it is being updated in https://github.com/cityofaustin/atd-data-tech/issues/10715